<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>240</horstretch>
    <verstretch>162</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>181</height>
    </rect>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Compras</string>
    </attribute>
    <widget class="QLineEdit" name="edt_idpro" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>27</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_nomepro" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>27</y>
       <width>371</width>
       <height>23</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>56</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>id_pro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>nome_pro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>63</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>qtd_pro</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_qtdpro" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4" >
     <property name="geometry" >
      <rect>
       <x>93</x>
       <y>63</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>vlr_unit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_vlrpro" >
     <property name="geometry" >
      <rect>
       <x>90</x>
       <y>80</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="inputMask" >
      <string/>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5" >
     <property name="geometry" >
      <rect>
       <x>232</x>
       <y>63</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>vlr_total</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_vlrtotal" >
     <property name="geometry" >
      <rect>
       <x>230</x>
       <y>80</y>
       <width>113</width>
       <height>23</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_localizar" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>120</y>
       <width>75</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Localizar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_comprar" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>120</y>
       <width>75</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Comprar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_sair" >
     <property name="geometry" >
      <rect>
       <x>190</x>
       <y>120</y>
       <width>75</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text" >
      <string>&amp;Sair</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ok" >
     <property name="geometry" >
      <rect>
       <x>280</x>
       <y>120</y>
       <width>75</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
    <widget class="QToolButton" name="btn_calc" >
     <property name="geometry" >
      <rect>
       <x>188</x>
       <y>76</y>
       <width>33</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset>../../../../../opt/kde/share/icons/crystalsvg/48x48/actions/forward.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2" >
    <attribute name="title" >
     <string>Tab 2</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3" >
    <attribute name="title" >
     <string>Page</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
